src/report_modules/report_module_github_ci/src/report_format_github_ci.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 CARIAD SE.
+ * Copyright 2024 BMW AG
+ *
+ * This Source Code Form is subject to the terms of the Mozilla
+ * Public License, v. 2.0. If a copy of the MPL was not distributed
+ * with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+#include "common/util.h"
+
+#include <memory>
+#include <xercesc/dom/DOM.hpp>
+#include <xercesc/parsers/AbstractDOMParser.hpp>
+#include <xercesc/parsers/XercesDOMParser.hpp>
+#include <xercesc/sax/ErrorHandler.hpp>
+#include <xercesc/sax/SAXException.hpp>
+#include <xercesc/sax/SAXParseException.hpp>
+#include <xercesc/util/ParseException.hpp>
+#include <xercesc/util/XMLString.hpp>
```

Comment:
I suspect this can be removed from the header. Optionally they can be moved in the respective compilation unit.

```suggestion
```